libs/MeshKernelApi/src/MeshKernel.cpp

```diff
@@ -988,21 +988,42 @@ namespace meshkernelapi
             }
             else
             {
+                throw meshkernel::AlgorithmError("Invalid MakeGridParameters");
```

Comment:
We should use something more descriptive in the error message rather than the struct name.